frontend/public/components/pod.tsx

```diff
@@ -104,15 +109,40 @@ const tableColumnClasses = [
   classNames('pf-m-hidden', { 'pf-m-visible-on-xl pf-u-w-10-on-2xl': showMetrics }),
   classNames('pf-m-hidden', { 'pf-m-visible-on-xl pf-u-w-10-on-2xl': showMetrics }),
   classNames('pf-m-hidden', 'pf-m-visible-on-2xl pf-u-w-10-on-2xl'),
+  classNames('pf-m-hidden', 'pf-m-visible-on-xl'),
+  classNames('pf-m-hidden', 'pf-m-visible-on-xl'),
+  classNames('pf-m-hidden', 'pf-m-visible-on-xl'),
```

Comment:
It seems like it should no longer be possible to use classes like `pf-m-visible-on-xl` for table columns. If I only select two columns, those columns should always be visible no matter what my screen width is.

Reply:
@spadgett good point...not sure what this css would be for this case...do we do this in other tables that you are aware of?

Reply:
> @spadgett good point...not sure what this css would be for this case...do we do this in other tables that you are aware of?

I think we need to cede control to the list view and let it calculate visibility based on screen width and the index of the visible columns. At each screen breakpoint, we show a certain number of columns. That has the potential to clean up the list views a ton since we can get rid of the CSS in most cases. You'd only need to set the CSS in special cases for when you need to control column width.